flutter/shell/platform/tizen/channels/platform_view_channel.cc

```diff
@@ -147,8 +149,13 @@ void PlatformViewChannel::OnCreate(
     if (focused_view) {
       focused_view->SetFocus(false);
     }
-    PlatformView* view =
-        iter->second->Create(*view_id, *width, *height, byte_message);
+#ifdef TV_PROFILE
+    double pixel_ratio = 1.6;
+#else
+    double pixel_ratio = 1.0;
```

Comment:
Why hard-coded?

Reply:
The reason why I hardcoded was that I did 1.6 with the fixed dpi value of 72 on the TV, and I didn't consider other pofiles yet, so the default value was 1.0.
However, I'll fix it in a way that calculates and uses it.